### ISSUE TITLE
[Resolved] Cannot determine the user who created an entry

### ISSUE THREAD

**Issue by KyferEz** [jasonrollins/shareplum#35]
Using Shareplum, I have successfully queried and extracted calendar events from SharePoint using the below sample code. However I need to know who created the calendar event, and cannot seem to find a method to determine this. 

Please if it can be done provide an example, and if not, then consider this an enhancement request to gather information on who created the entry.
```python
site = Site('http://sharepoint.site.net/sites/TS/Network/', auth=cred)
calendar = site.List('PTO')
query = {'Where': ['And', ('Geq', 'Start Time', datetime.datetime(2018, 10, 1, 0, 0)),
                              ('Lt', 'Start Time', datetime.datetime(2019, 12, 1, 0, 0)),
                                 ]}
fields = None
sp_data = calendar.GetListItems(query=query, fields=fields)
```

**Comment by KyferEz** [comment]
@jasonrollins Can you please look at this issue when you have a moment? Is there a way to determine the user who created an event on a shared calendar?

**Comment by fatfingers23** [comment]
Not sure what fields you are passing over, but i tested with the fields "Titled" and "Created By"(which seems to be what you are looking for) by doing the below. Only thing i did that was different from yours i can tell is i specified "Created By" in Fields and looks to be working fine.
 I think maybe you were missing that?
```
sp_data = list.GetListItems(fields=['Created By', 'Title'], query= query)
```

**Comment by KyferEz** [comment]
@fatfingers23 Thank you! I was using None in the fields hoping to get all fields because I didn't know what fields were available. It seemed to be working as I got numerous fields, but obviously was not giving me all fields because the Created By field didn't show. 

On that note, if anyone else has this issue and doesn't know how to get the fields, I just located some info (Sharepoint 2010):
http://blog.softartisans.com/2009/12/08/list-of-internal-names-for-sharepoint-fields/
https://social.msdn.microsoft.com/Forums/office/en-US/75ca6fab-56f3-4bf4-aae0-2d29821778a2/how-to-get-internal-names-of-columns-in-sharepoint-lists?forum=sharepointdevelopmentlegacy
